gradle/gradle-mvn-push.gradle

```diff
@@ -11,22 +8,58 @@ def getSnapshotRepositoryUrl() {
             : "https://oss.sonatype.org/content/repositories/snapshots/"
 }
 
-apply plugin: "com.vanniktech.maven.publish"
+apply plugin: "maven-publish"
 
-publishing {
-    repositories {
-        maven {
-            url = version.endsWith('SNAPSHOT') ? getSnapshotRepositoryUrl() : getReleaseRepositoryUrl()
+android {
+    publishing {
+        singleVariant("release") {
+            withSourcesJar()
+            withJavadocJar()
         }
     }
 }
 
-mavenPublish {
-    releaseSigningEnabled false
-}
+afterEvaluate {
+    publishing {
+        repositories {
+            maven {
+                url = version.endsWith('SNAPSHOT') ? getSnapshotRepositoryUrl() : getReleaseRepositoryUrl()
+            }
+        }
 
-project.afterEvaluate {
-    project.tasks.named("androidSourcesJar").configure {
-        from project.android.sourceSets.main.java.srcDirs
+        publications {
+            release(MavenPublication) {
+                afterEvaluate {
+                    from components.release
+                }
+                groupId = "com.badoo.ribs"
+                version = VERSION_NAME
+
+                pom {
+                    name.set("MVICore")
+                    description.set("Badoo RIBs is an evolution of Uber RIBs, with notable differences and additions")
+                    url.set("https://github.com/badoo/RIBs/")
+                    licenses {
+                        license {
+                            name.set("The Apache Software License, Version 2.0")
+                            url.set("https://www.apache.org/licenses/LICENSE-2.0.txt")
+                        }
+                    }
+                    developers {
+                        developer {
+                            id.set("badoo")
+                            name.set("Badoo Trading Limited")
+                            email.set("mvicore@team.bumble.com")
```

Comment:
Remove email? Looks like we did not have it before.

Reply:
weird. I swear I copied it from somewhere. will remove